timemachine/fe/rbfe.py

```diff
@@ -359,19 +360,17 @@ def optimize_coordinates(initial_states, min_cutoff=0.7) -> List[NDArray]:
 
 
 def estimate_relative_free_energy(
-    mol_a,
-    mol_b,
+    mol_a: Chem.rdchem.Mol,
+    mol_b: Chem.rdchem.Mol,
     core: NDArray,
     ff: Forcefield,
     host_config: Optional[HostConfig],
     seed: int,
-    n_frames: int = 1000,
     prefix: str = "",
     lambda_interval: Optional[Tuple[float, float]] = None,
     n_windows: Optional[int] = None,
     keep_idxs: Optional[List[int]] = None,
-    n_eq_steps: int = 10000,
-    steps_per_frame: int = 400,
+    md_params: Optional[MDParams] = None,
```

Comment:
nit: if `MDParams` were defined with `@dataclass(frozen=True)`, might be cleaner to just give a default instance instead of making it optional?

Reply:
Changed in https://github.com/proteneer/timemachine/pull/1026/commits/b18d1e0bd6b82a41b9b20f77182656b126502f97

And changed defaults to what they had been previously rather than what we default to for production runs. 400 steps per frame, 1000 frames, 10k equil